features/performance/test/featuregate_test.go

```diff
@@ -0,0 +1,35 @@
+package performance_test
+
+import (
+	"bytes"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/util/yaml"
+
+	configv1 "github.com/openshift/api/config/v1"
+)
+
+// beware the typo (sic)
```

Comment:
why not fixing the typo? :)

Reply:
Yup, in a separate PR for better visibility and awareness